lib/src/core/toastification.dart

```diff
@@ -218,9 +218,9 @@ class Toastification {
     ToastificationAnimationBuilder? animationBuilder,
     ToastificationType? type,
     ToastificationStyle? style,
-    required String title,
+    Widget? title,
```

Comment:
I'm not sure about making title nullable for now. 

Do you have any reason for that?

Reply:
Maybe someone doesn't want to have a title for example have extra space for the description.

Reply:
To clarify Nullable is commonly used by material and Cupertino devs when it comes to building U.I components it provides greater Flexibility to the dev.

In this case, this allows us to provide more vertical space for the description field, or/and gain a little performance by not having to initialize an empty text Widget.

Reply:
Yes, you are right